src/main/webapp/WEB-INF/tags/instructor/home/courseTable.tag

```diff
@@ -3,57 +3,62 @@
-            <th>Status</th>
-            <th>
-                <span class="text-nowrap" title="<%= Const.Tooltips.FEEDBACK_SESSION_RESPONSE_RATE %>" 
-                      data-toggle="tooltip" data-placement="top">Response Rate</span>
-            </th>
-            <th class="no-print">Action(s)</th>
-        </tr>
-    </thead>
-    <c:if test="${empty sessionRows}">
-        <tr>
-            <td>
-                <span class="text-muted">
-                    This course does not have any sessions yet.
-                </span>
-            </td>
-            <td></td>
-            <td></td>
-            <td></td>
-            <td></td>
-            <td></td>
-        </tr>
-    </c:if>
-    <c:forEach items="${sessionRows}" var="sessionRow" varStatus="i">
-        <tr id="session${i.index}">
-            <td>
-                ${sessionRow.name}
-            </td>
-            <td class="text-nowrap">
-                <span title="${sessionRow.startTimeToolTip}" data-toggle="tooltip">${sessionRow.startTime}</span>
-            </td>
-            <td class="text-nowrap">
-                 <span title="${sessionRow.endTimeToolTip}" data-toggle="tooltip">${sessionRow.endTime}</span>
-            </td>
-            <td>
-                <span title="${sessionRow.tooltip}" data-toggle="tooltip" data-placement="top">
-                    ${sessionRow.status}
-                </span>
-            </td>
-            <td class="session-response-for-test">
-                <a oncontextmenu="return false;" href="${sessionRow.href}">Show</a>
-            </td>
-            <td class="no-print text-nowrap padding-right-25px">
-                <tif:feedbackSessionActions actions="${sessionRow.actions}" />
-            </td>
-        </tr>
-    </c:forEach>
-</table>
\ No newline at end of file
+<div class="panel-collapse collapse">
+    <div class="panel-body padding-0">
+        <table class="table-responsive table table-striped table-bordered">
+            <thead>
+                <tr>
+                    <th onclick="toggleSort(this);" class="button_sortname button-sort-none">
+                        Session Name<span class="icon-sort unsorted"></span>
+                    </th>
+                    <th onclick="toggleSort(this,instructorHomeDateComparator);" class="button_sortstartdate button-sort-none">Start Date<span class="icon-sort unsorted"></span></th>
+                    <th onclick="toggleSort(this,instructorHomeDateComparator);" class="button_sortenddate button-sort-none">End Date<span class="icon-sort unsorted"></span></th>
+                    <th>Status</th>
+                    <th>
+                        <span class="text-nowrap" title="<%= Const.Tooltips.FEEDBACK_SESSION_RESPONSE_RATE %>" 
+                            data-toggle="tooltip" data-placement="top">Response Rate</span>
```

Comment:
Hmm I must have missed this earlier: this line should be indented 8 spaces instead.

As I believe requesting for this change might cause some confusion, allow me to clarify:

Lines that are supposed to represent content or inner elements of an element (such as the `Session Name...` line in line 12) should be indented 4 spaces. However, lines that are a continuation of the previous one (eg. this line, which is a continuation of the `span` element on the previous line) should be indented 8 spaces, to distinguish it from content/inner elements.